platform/linuxbsd/wayland/dynwrappers/wayland-client-core.c

```diff
@@ -0,0 +1,607 @@
+// This file is generated. Do not edit!
+// see https://github.com/hpvb/dynload-wrapper for details
+// generated by ./generate-wrapper.py 0.3 on 2022-10-24 09:08:22
+// flags: ./generate-wrapper.py --include /usr/include/wayland-client-core.h --sys-include <wayland-client-core.h> --soname libwayland-client.so --init-name wayland_client --output-header wayland-client-core.h --output-implementation wayland-client-core.c
```

Comment:
Probably should be `libwayland-client.so.0`, e.g., https://packages.debian.org/sid/amd64/libwayland-client0/filelist